bioimageio/core/_resource_tests.py

```diff
@@ -110,7 +133,16 @@ def load_description_and_test(
         else:
             weight_formats = [weight_format]
         for w in weight_formats:
-            _test_model_inference(rd, w, devices, decimal)
+            # Note: new_precision_args is created like this to avoid type check errors
```

Comment:
I think directly providing `absolute_tolerance` and  `relative_tolerance` to `_test_model_inference` would be a bit more straight forward...

Reply:
Thoughts in this [comment](https://github.com/bioimage-io/core-bioimage-io-python/pull/418#discussion_r1738611678), I can make the change